### ISSUE TITLE
Idea: Utilize as a "Now Playing" button

### ISSUE THREAD

**Issue by rabbly** [polywock/globalSpeed#275]
The add-in already acts similarly to a "Now Playing" button - it shows the currently playing media. However when you pause that media it will be removed from the drop down if you close the menu and re-open.

Since Edge doesn't support the standard Chromium "Now Playing" feature (shown in screenshot below in the Brave browser) it would be great if globalSpeed could handle this. It seems like it would be a small enhancement to do so.

![image](https://user-images.githubusercontent.com/2348746/197291112-39bb6982-6f68-463d-8a16-c6818ad42a37.png)

**Comment by polywock** [comment]
Hello. Near the media controls, there is a little pin icon. If a video/audio is selected, it will be prioritized when using Global Speed's media shortcut keys. Useful when wanting to control video's in background tabs, or even with different applications with global shortcut keys ([example](https://www.youtube.com/watch?v=EZA-TnVAyDo)). 

It should also keep the video/audio visible even when paused, which sounds like is what you need.  

![img1](https://user-images.githubusercontent.com/31208859/197303960-d1b9d49e-f45a-4fee-b16a-b37770122a37.png)

**Comment by rabbly** [comment]
Whilst I can confirm that does work, it seems like an unnecessary step. Wouldn't it be a useful improvement to retain paused media in the list like the Chrome "now playing" does?

**Comment by polywock** [comment]
I started working on my media list before I knew about the Now Playing feature (which was in beta at the time). I only made so users can select which video they want to prioritize for the shortcut keys. I could mimic Now Playing, but I see some issues. On a subreddit like r/videos, there can be dozens of videos on a single page.  The Now Playing feature can access internal data so they could more easily sort what's important and not, so they only show the single video visible on the screen. For extensions to the same, it will much more difficult to calculate this information. 

I will look into improving it.
